src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -939,8 +945,14 @@ def forward(
         )
         return_dict = return_dict if return_dict is not None else self.config.use_return_dict
 
+        # Cast the input to the expected `dtype`
+        expected_input_dtype = self.embeddings.patch_embedding.weight.dtype
+        if expected_input_dtype != pixel_values.dtype:
+            pixel_values = pixel_values.to(expected_input_dtype)
```

Comment:
Same here

Reply:
Fixed in [57c4271](https://github.com/huggingface/transformers/pull/20411/commits/57c427152f4552666c1e3bb2d5975084c8432ef9)